src/client/lobby/lobby.js

```diff
@@ -3,16 +3,31 @@ import React, { Component } from 'react';
 import config from '../config/config';
 
 export default class Lobby extends Component {
-    componentWillMount() {
-        cloak.configure({
-            // TODO: Add configuration
-        });
+    constructor(props) {
+        super(props);
+        this.state = {
+            title: "Ready",
+            ready: false
+        };
+        this.onClick = this.onClick.bind(this);
+    }
 
-        cloak.run(config.cloakAddress);
+    onClick(e) {
+        const ready = !this.state.ready;
+        const title = this.state.ready ? "Ready" : "Unready";
```

Comment:
I'd make ready and unready consts at the top of the file if you're going to do this. Or, you can write your button like this:

`<button onClick={this.onClick}>{this.state.ready? "Ready" : "Unready" }</button>`

and you don't need the title in state at all, and the strings make sense because they're just in the markup.